apps/frontend/src/components/global/TopbarDropdown.vue

```diff
@@ -29,29 +29,29 @@
               text="Admin Panel"
               icon="mdi-shield-account"
               link="/admin"
-              >Admin Panel</IconLinkItem
-            >
+              >Admin Panel
```

Comment:
i really dislike when content has a lingering brace on it

i.e.

```
<a
>inner stuff
</a>
```

should be

```
<a>
inner stuff
</a>
```